dev/az/create-secret-manager-secret.yaml

```diff
@@ -0,0 +1,33 @@
+---
+secrets:
+  - secretName: jade-sa
+    vals:
+      - kubeSecretKey: datareposerviceaccount
+        path: secret/dsde/datarepo/dev/sa-key-b64
+        encoding: base64
+        vaultKey: sa
+  - secretName: database-pwd-az
+    vals:
+      - kubeSecretKey: datarepopassword
+        path: secret/dsde/datarepo/dev/helm-datarepodb-dev
+        vaultKey: datarepopassword
+      - kubeSecretKey: stairwaypassword
+        path: secret/dsde/datarepo/dev/helm-datarepodb-dev
+        vaultKey: stairwaypassword
+  - secretName: azure-az
+    vals:
+      - kubeSecretKey: applicationsecret
+        path: secret/dsde/datarepo/dev/helm-azure
+        vaultKey: applicationsecret
+      - kubeSecretKey: synapse-us-east-sql-admin-user
+        path: secret/dsde/datarepo/dev/helm-azure
+        vaultKey: synapse-us-east-sql-admin-user
+      - kubeSecretKey: synapse-us-east-sql-admin-password
+        path: secret/dsde/datarepo/dev/helm-azure
+        vaultKey: synapse-us-east-sql-admin-password
+  - secretName: rbs-sa-az
+    vals:
+      - kubeSecretKey: rbsserviceaccount
+        path: secret/dsde/terra/kernel/dev/dev/buffer/client-sa
```

Comment:
Would we want to use tools env here instead? … or I suppose it shouldn't matter soon, if we're only creating azure resources!

Reply:
That's a good point! It doesn't matter here really, but I changed it to tools

Reply:
I noticed several other personal dev environments use dev instead of tools so I'll make a note to change that in a follow-on PR

Reply:
I think we've kinda left it up to people as to which one to use.. I've set mine to tools since I've used it mostly for running tests against, but it could make sense for people to want their dev projects to create projects that will stick around for a few days as demos/etc.